### ISSUE TITLE
Collector "number" isn't a number

### ISSUE THREAD

**Issue by skarsol** [gaddlord/mtg.studio#529]
Trying to do an export I see that there is no standard format for the Collector Number field and instead it contains an exact copy of what appears on the card, including unneeded extra data. Some example values are "125/284", "149/168 R", "240 R", "R 0130". What is this intended to be used for? You can't use it to sort, you can't use it to export/import with other tools, it's more (and in the cases of the letter, duplicative) info than needed to determine uniqueness. Why not just have it be the actual collector number (in the examples above, 125, 149, 240, 130)?

This is especially egregious in exports because the name field can't be used for imports due to the metadata that has been added to the name.

**Comment by gaddlord** [comment]
Lets break this down. I believe we are talking about a combination of a existing feature and a couple of feature requests.
Existing feature: MTG Studio lists in the Collector No exactly what is written on the card.

New feratures requested:
1. Sorting by Collect No to sort the column by the X value of "[Rarity] X/[Cards in Set][a|b|c|d|] [Promo SUffix]"
2. Being able to export the X value instead of the whole Collector No line value.

Am I getting it right?

**Comment by skarsol** [comment]
Yes, I'd like to have just the number available in a column by itself for sorting/import/export purposes. Collector Number vs Collector String effectively.

**Comment by gaddlord** [comment]
Sorting now works with X value from "prefix X/Y suffix." in version 3.224

**Comment by texasaggie1994** [comment]
Seems to be a regression in 3.228. Sorting for with the rarity in front of the number appears to mess up the sort.  

<img width="633" height="815" alt="Image" src="https://github.com/user-attachments/assets/37a90fb4-3d82-4461-bd5e-8d4ef26d657e" />
